bpf/pid.h

```diff
@@ -6,15 +6,16 @@
 #include "bpf_core_read.h"
 #include "pid_types.h"
 
-#define MAX_CONCURRENT_PIDS 3000 // estimate: 1000 concurrent processes (including children) * 3 namespaces per pid
+#define MAX_CONCURRENT_PIDS 3001 // estimate: 1000 concurrent processes (including children) * 3 namespaces per pid
```

Comment:
In another PR, I'd make this value configurable by the user.

Here is an example about how we do this for the Network flows:
https://github.com/grafana/beyla/blob/a22db7476b098f4063ff42f4563667bdf3f1d91a/bpf/flows_common.h#L39-L45

https://github.com/grafana/beyla/blob/a22db7476b098f4063ff42f4563667bdf3f1d91a/pkg/internal/netolly/ebpf/tracer.go#L85-L86

Reply:
Cool!